frontend/src/components/Header.vue

```diff
@@ -69,6 +68,9 @@ export default {
   color: #fff;
   background: none;
 }
+.gov-header .v-btn.title{
+  text-transform: none;
```

Comment:
Do we need to explicitly specify no text-transform? None is the default value. Is this needed to override something with a lower specificity?

Reply:
yeah this is odd but i think v-btn default is uppercase. https://vuetifyjs.com/en/components/buttons

Reply:
Ah right, yeah that makes sense.
However, I think we'd want to stick with Veutify default styling if possible as it's an implementation of Material Design. Could check with Mina if it's important to deviate from the Veutify styles. If so we might want to look at setting global styles, as we want the app to be consistent. (ie, if this button is not capitalized, why are all the others in the app).

Reply:
agreed. i'm leaving it as default vuetify style. using a global stylesheet later sounds like a great idea